src/components/AddBookForm.css

```diff
@@ -0,0 +1,29 @@
+input {
+  border: solid 1px whitesmoke;
+  background: white;
+  padding: 10px;
+  margin-right: 20px;
+}
+
+.categoryInput {
+  border: solid 1px whitesmoke;
+  background: white;
+  padding: 10px;
+  margin-right: 20px;
+}
+
+form h2 {
+  margin: 0.8em 0;
+  color: rgb(87, 87, 87);
+}
+
+form button {
+  background-color: #4395fc;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  padding: 10px 20px;
```

Comment:
- [ ] For DRY CSS kindly combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

Reply:
Thank you for your time and for the tips, I'm going to apply them